include/isa.h

```diff
@@ -9,4 +10,9 @@
  */
 struct CPUState process_instruction(struct CPUState state);
 
+#define N 31
```

Comment:
Given that everything is global namespace in C, it would be nice if you rename these to CPSR_N etc. to avoid conflicts